packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -354,6 +354,13 @@ export interface GlobalSecondaryIndexProps extends SecondaryIndexProps, SchemaOp
    * @default 5
    */
   readonly writeCapacity?: number;
+
+  /**
+   * Whether CloudWatch contributor insights is enabled.
+   *
+   * @default false
+   */
+  readonly contributorInsightsEnabled?: boolean;
```

Comment:
Is this the correct scope for adding this? It seems like a prop that should be available on more than just a global secondary index.

Reply:
Thanks for noting this! I will move this option into `SchemaOptions`